### ISSUE TITLE
About GOT-10k test set results

### ISSUE THREAD

**Issue by suyeH** [researchmm/Stark#8]
Hi, Thanks for your wonderful work. I notice that [Transformer Tracking](https://arxiv.org/abs/2103.15436) use the model trained with all datasets(LaSOT, GOT10K, COCO, TrackingNet) to get the evaluation result on  GOT-10k test set, and the result is much better than the model trained with GOT10K only. 

![image](https://user-images.githubusercontent.com/23137816/120223744-a5e25300-c274-11eb-9d35-fcd35c340364.png)

However, when I use the STARK-S50 pre-trained model(trained on all datasets) in your model zoo to evaluate the GOT-10k test set, I find that the AO is 0.688, which only gains small improvement compared with 0.672

I am confused with this phenomenon. Have you ever tried to evaluate the model trained with all datasets on GOT-10k test set? Or can you kindly explain the reason why there is just little performance gain to use the model trained on all datasets?

**Comment by MasterBin-IIAU** [comment]
Hi, thank you for your interest in our work. 
You can try evaluating the STARK-ST50 model trained with full datasets on GOT-10K. I remember that the result is also around 0.72. Please feel free to try.

**Comment by suyeH** [comment]
> Hi, thank you for your interest in our work.
> You can try evaluating the STARK-ST50 model trained with full datasets on GOT-10K. I remember that the result is also around 0.72. Please feel free to try.

Thanks. I've tried STARK-ST50, and the result is fine. But I'm still confused about the reason why the STARK-S50 model trained with all dataset gains only minor improvement compared with GOT10K only but  STARK-ST50 will not. Do you have any idea about this?

**Comment by MasterBin-IIAU** [comment]
Hi, I think a potential reason might be that STARK-S lacks a template update mechanism and this restricts its discriminative ability, especially when there are similar distractors. In this case, feeding model with more data shall not bring too much improvement.

**Comment by suyeH** [comment]
> Hi, I think a potential reason might be that STARK-S lacks a template update mechanism and this restricts its discriminative ability, especially when there are similar distractors. In this case, feeding model with more data shall not bring too much improvement.

This may be the reason. Thanks a lot for your reply.
